0800-appendix-198236-trib-to-kitwanga.Rmd

```diff
@@ -23,7 +23,9 @@ my_site <- 198236
 ## Site Location {.unnumbered}
 
 PSCIS crossing `r as.character(my_site)` is located on
-`r fpr::fpr_my_pscis_info()`. The site is located on `r fpr_my_pscis_info(col_pull = road_name)`.
+`r fpr::fpr_my_pscis_info()`, on the `r fpr_my_pscis_info(col_pull = road_name)`. This crossing was located
+`r round(fpr::fpr_my_bcfishpass(col_pull = downstream_route_measure)*0.001, 1)`km
+upstream from the confluence with the Kitwanga River. This river drains into Kitwanga Lake (a.k.a Kitwancool Lake or Gitanyow Lake). The underlying territory of this area belongs to the Gitanyow, an Indian reserve community of the Gitxsan first nations.  
```

Comment:
"The underlying territory of this area belongs to the Gitanyow, an Indian reserve community of the Gitxsan first nations."

the Gitanyow are not an "Indian reserve community". 

Gitanyow is an First Nations community of the Gitxsan people (https://www.bcafn.ca/first-nations-bc/north-coast/gitanyow) .  The Gitanyow peoples are known collectively as the Gitanyow Nation. The Gitanyow Nation have a history that is many thousands of years older than the imposed colonial reserve system with adawaak – "true traditions"; the ancient histories passed down by oral tradition. whoa. buddy

https://www.gitanyowchiefs.com/images/uploads/land-use-plans/Gitanyow-R-R-Agreement-2012.pdf
https://www.gitanyowchiefs.com/wilp-system/
https://www.gitanyowchiefs.com/about/history/